tests/e2e/matchers/to-have-tracking.js

```diff
@@ -20,7 +20,6 @@
  * External dependencies
  */
 import { getDefaultOptions } from 'expect-puppeteer';
-import { Page, ElementHandle } from 'puppeteer';
```

Comment:
Why was this removed here (also in the other file)? Shouldn't we import them so that the reference in the docs below is interpreted correctly?

Reply:
> Why was this removed here (also in the other file)?

Eslint blames us that these are unused vars. So, i have decided to remove it (since technically speaking - it's not needed there) instead of disabling eslint check for that line.

> Shouldn't we import them so that the reference in the docs below is interpreted correctly?

Do we maintain publicly available documentation for this? Do we really need it to be interpreted correctly?

Reply:
Done.